doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -378,7 +378,19 @@ Bluetooth Fast Pair samples
 Bluetooth Mesh samples
 ----------------------
 
-|no_changes_yet_note|
+* Added support for the :ref:`zephyr:nrf54l15dk_nrf54l15` board in the following samples:
+
+  * :ref:`bluetooth_mesh_sensor_client` sample
+  * :ref:`bluetooth_mesh_sensor_server` sample
+  * :ref:`bluetooth_ble_peripheral_lbs_coex` sample
+  * :ref:`bt_mesh_chat` sample
+  * :ref:`bluetooth_mesh_light_switch` sample
+  * :ref:`bluetooth_mesh_silvair_enocean` sample
+  * :ref:`bluetooth_mesh_light_dim` sample
+  * :ref:`bluetooth_mesh_light` sample
+  * :ref:`ble_mesh_dfu_target` sample
+  * :ref:`bluetooth_mesh_light_lc` sample
+  * :ref:`ble_mesh_dfu_distributor` sample
```

Comment:
```suggestion
* Added support for the :ref:`zephyr:nrf54l15dk_nrf54l15` board in the following samples:

  * :ref:`bluetooth_mesh_sensor_client`
  * :ref:`bluetooth_mesh_sensor_server`
  * :ref:`bluetooth_ble_peripheral_lbs_coex`
  * :ref:`bt_mesh_chat`
  * :ref:`bluetooth_mesh_light_switch`
  * :ref:`bluetooth_mesh_silvair_enocean`
  * :ref:`bluetooth_mesh_light_dim`
  * :ref:`bluetooth_mesh_light`
  * :ref:`bluetooth_mesh_light_lc`
  * :ref:`ble_mesh_dfu_target`
  * :ref:`ble_mesh_dfu_distributor`
```

Reply:
But in 2.7.0 release we used word `sample` at the end:
https://github.com/nrfconnect/sdk-nrf/blob/0a59bda839b042c08ec52158a13653ec1f4bdeae/doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst?plain=1#L641-L653

Reply:
I simply copy-pasted that section

Reply:
It's not a cardinal error to leave it. But IMO it would be good to remove the repetitive 'sample' word, especially when it's already established in the introductory sentence that this is done in 'the following samples'.
And, things sometimes slip for us, we don't always catch all that could be improved every time. But once spotted, probably ok to do something about it :)

Reply:
Updated